configs/docker-release.toml

```diff
@@ -23,3 +23,7 @@ model_type = "M3"
 
 [model]
 size = 4
+
+[metrics]
+store_url = "http://influxdb:8086"
```

Comment:
I suspect that some settings will be specific to the metrics backend we're using. Maybe we have different settings for each backend:

```toml
[metrics.influxdb]
url = "http://influxdb:8086"
store_name = "metrics"
```

Reply:
fair point, I think namespacing `[metrics.influxdb]` is indeed useful